docs/source/_json/addons_get_list.resp

```diff
@@ -1,3 +1,51 @@
-GET /plone/@addons HTTP/1.1
-Accept: application/json
-Authorization: Basic YWRtaW46c2VjcmV0
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+  "@id": "http://localhost:55001/plone/@addons",
+  "batching": {
+    "@id": "http://localhost:55001/plone/@addons",
+    "first": "http://localhost:55001/plone/@addons?b_start=0",
+    "last": "http://localhost:55001/plone/@addons?b_start=1500",
+    "next": "http://localhost:55001/plone/@addons?b_start=25"
+  },
+  "items": [
+    {
+      "description": "Optional plone.session refresh support.",
```

Comment:
Each item should also get an `@id` property with the URL for the individual addon record (same for the response for individual addons).
E.g. `"@id": "http://localhost:55001/plone/@addon/plone.session"`